src/main/java/HookKiller/server/board/entity/Board.java

```diff
@@ -0,0 +1,41 @@
+package HookKiller.server.board.entity;
+
+import HookKiller.server.board.type.BoardType;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+import java.util.List;
+
+/**
+ * id : PK
+ * name :게시판 명
+ * boardType : 게시판 종류
+ * description : 게시판 사용 용도
+ */
+
+@Entity
+@Getter
+@Table(name = "Board")
```

Comment:
테이블명칭만  아래와 같이 설정 바람.
```suggestion
@Table(name = "tbl_board")
```